docs/production.md

```diff
@@ -0,0 +1,155 @@
+# Running an instance of the Research Software Directory in production
+
+## Temporarily disabling the admin interface
+
+You can stop users from making changes to the database by disabling the authentication to the admin interface, as follows:
+
+```shell
+docker-compose stop auth
+```
+
+This can be useful when upgrading the data or software to a newer version.
+
+Note that even with a stopped `auth` service, determined users can still access `backend` directly, and users who were
+logged in before you disable `auth` will still be able to use the admin interface to make changes.
+
+It it recommended that you post a message that users see when they try to use the admin interface. The easiest way to do
+this is as follows:
+
+```shell
+# check volume names
+docker volume ls
+
+# pick the admin interface volume (probably rsd_static_admin)
+docker volume inspect rsd_static_admin
+
+# check where it is mounted
+sudo -i
+cd <to the where the admin static files are>
+
+# rename index.html
+mv index.html index.html.disabled
+
+# put a message in index.html
+echo "<html>" \
+"<head></head>" \
+"<body>Sorry, we're doing maintenance right now. " \
+"Hopefully be back soon.</body></html>" > index.html
+```
+
+You can check if it works by using your browser to navigate to the admin interface. Instead of the normal interface, you
+should now see your message.
```

Comment:
Please explain how to revert back

```suggestion
should now see your message.

To enable admin interface again do the previous instructions in reverse

```shell
# inside admin container
mv /build/index.html /build/index.html.disabled

# Enable logins again
docker-compose start auth
```

```

Reply:
committed your suggestion (I'll flip the order of the `mv` arguments in a subsequent commit)